Week3/homework/js-exercises/1_GiveCompliment.js

```diff
@@ -0,0 +1,13 @@
+'use strict'
+function giveCompliment(yourName) {
+    let complimentWords = ['great', 'wonderful', 'amazing', 'majestic', 'lovely', 'kind', 'stunning', 'awesome', 'elegant', 'superb'];
+    let randomCompliment = complimentWords[Math.floor((Math.random()*complimentWords.length))];
+    return 'You are ' + randomCompliment + ' ' + yourName + '!';
+
```

Comment:
`const` instead of `let` here please. Also, would like to introduce a new concept in JavaScript called template literals - https://developer.mozilla.org/nl/docs/Web/JavaScript/Reference/Template_literals

Using this, you need not use + operator for concatenating strings.